src/FGFDMExec.cpp

```diff
@@ -154,7 +154,7 @@ FGFDMExec::FGFDMExec(FGPropertyManager* root, unsigned int* fdmctr) : Root(root)
     Allocate();
   } catch (const string& msg ) {
     cout << "Caught error: " << msg << endl;
```

Comment:
I was just curious since all the others were removed.  If it's fine tell me, and I'll merge.